contrib/enum_metrics/enum_metrics/errant_enums.py

```diff
@@ -0,0 +1,191 @@
+import sys
+import argparse
+import json
+import dbclient as db
+import esclient as es
+import config as cf
+
+LOCALHOST = 'localhost'
+QE01 = 'qe01'
+QE02 = 'qe02'
+STAGING = 'staging'
+PROD = 'prod'
+
```

Comment:
@drenalin23  I just made the script environment agnostic. It basically works with whatever environment we configure in config.ini.